corehq/apps/reports/templates/reports/form/partials/single_form_tree.html

```diff
@@ -7,6 +7,12 @@
                     <strong>{% trans "Hidden Values (Form Calculations)" %}</strong>
                 </td>
             </tr>
+        {% elif 'commcare_usercase/' in question.value %}
+            <tr class="info">
+                <td colspan="2">
+                    <strong>{% trans "User Case Data" %}</strong>
+                </td>
+            </tr>
```

Comment:
In a normal form, it seems to me that normal case updates are not shown at all. In this example the questions "fav_color", "under_bed", "in_closet" and "in_head" all update the "kid" case, but only the question responses are shown in the form details, not the case updates.

In an advanced form the case updates are shown, and they fall into the next section, "Unrecognized Data". Here's an advanced form that updates both a user case and a normal "kid" case:
![unrecognized_data](https://cloud.githubusercontent.com/assets/708421/8127848/24c50fc4-10fa-11e5-9c9c-47658219064a.png)